internal/metrics/metrics.go

```diff
@@ -30,15 +39,47 @@ func New() Metrics {
 			Help: "Number of existing KMMO Modules",
 		},
 	)
-	return &metrics{kmmoResourcesNum: kmmoResourcesNum}
+	completedStages := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: completedKMMOStageQuery,
+			Help: "For a given kmmo,namespace, stage(device-plug, driver-container, build), 1 if the stage is completed, 0 if it is not.",
+		},
+		[]string{"kmmo", "namespace", "stage"},
+	)
+
+	nodesUtilization := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: nodesUtilizationQuery,
+			Help: "Nodes that driver-container/device-plugin shoud run on based on selector, nodes they will run based on kernel mappings, currently deploed nodes",
```

Comment:
I think have highly variable labels will lead to huge cardinality. If we want to to expose a list of nodes to the user, maybe the `Module`'s status is a better place?

Reply:
agree, removing this metric, will open a PR for adding the info to the mod.Status